src/components/NavBarSite.jsx

```diff
@@ -4,20 +4,52 @@ import Nav from "react-bootstrap/Nav";
 import Navbar from "react-bootstrap/Navbar";
 import Container from "react-bootstrap/Container";
 import { LinkContainer } from "react-router-bootstrap";
+import getArticles from "../assets/Utils/getArticles";
 
-const NavBarSite = () => {
+const NavBarSite = ({ topics }) => {
+  function handleTopic() {
```

Comment:
You haven't defined e in the function here.
Making me think that the function isn't doing anything,.
It also doesn't return anything, what is it supposed to be doing? isn't the api call made elsewhere for the articles?